src/Sulu/Bundle/AdminBundle/Resources/js/components/Checkbox/Checkbox.js

```diff
@@ -0,0 +1,54 @@
+// @flow
+import React from 'react';
+import classnames from 'classnames';
+import Icon from '../Icon';
+import checkboxStyles from './checkbox.scss';
+
+export default class Checkbox extends React.PureComponent {
+    props: {
+        checked: boolean,
+        skin: 'dark' | 'light',
+        className: string,
+        onChange?: (checked: boolean) => void,
+    };
+
+    input: HTMLInputElement;
+
+    static defaultProps = {
+        skin: 'dark',
+        className: '',
+    };
+
+    handleChange = (event: Event) => {
+        if (this.props.onChange && typeof event.target.checked === 'boolean') {
```

Comment:
It works for me if I use `event.currentTarget` (which I think is the correct one anyway).